docs/dev.md

```diff
@@ -125,6 +125,11 @@ Start a Liberty server in a local container using the Dockerfile that you provid
 
 When dev mode runs with container support, it builds a container image and runs the container. You can examine the commands that it uses to build and run the container by viewing the console output of dev mode. Additionally, it still provides the same features as the `dev` goal. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. target/liberty/wlp/usr/servers/defaultServer/logs.
 
+N.B. starting in 3.6, dev mode invokes `generate-features`. Ensure that the `generated-features.xml` configuration file is copied via your Dockerfile.
+```dockerfile
+COPY --chown=1001:0  target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/generated-features.xml /config/configDropins/overrides/
```

Comment:
Suggestion: another example that says "On Linux copy the entire `overrides` directory." and the appropriate sample COPY command.

Reply:
Good idea, I've added this.